samples/highcharts/blog/blog-update-options/demo.css

```diff
@@ -0,0 +1,127 @@
+.container {
```

Comment:
I think all CSS rules should be namespaced. I see in the utils that the styles of the demo affects the rest of the page, and we risk the same on the web page:

`.highcharts-demo h5 ...`
`.highcharts-demo .container.controls ...`